src/components/common/Detail.vue

```diff
@@ -11,10 +11,10 @@
     <div :class="{ 'invalid-section': invalidSection }">
       <v-card flat class="detail-card">
         <v-row no-gutters>
-          <v-col cols="2" xs="3">
+          <v-col cols="3" sm="2">
```

Comment:
Lint tried to change this to `<v-col class="xs" cols="2">` but that's incorrect.

As it turns out, the "xs" property doesn't exist for v-cols. [ref](https://v2.vuetifyjs.com/en/components/grids/#caveats)

So I updated it this way, which is nicely responsive (the cols stack below sm size).